templates/layouts/backend.tpl

```diff
@@ -182,10 +182,20 @@
 									{foreach from=$breadcrumbs item="breadcrumb" name="breadcrumbs"}
 										<li>
 											{if $smarty.foreach.breadcrumbs.last}
-												<span aria-current="page">{$breadcrumb.name|escape}</span>
+												<span aria-current="page">
+													{if $breadcrumb.unscapeHtml}
```

Comment:
I think there's a bit of spaghetti in the use of unescapes on breadcrumbs. Better instead to introduce a `format` property to each `$breadcrumb` entry, defaulting to plaintext, but also supporting HTML. Please make sure that we're also applying `stripUnsafeHtml` as needed -- just applying `unescape`, at a glance, risks allowing anything through.

Reply:
updated .